src/zjs_timers.h

```diff
@@ -5,5 +5,6 @@
 
 void zjs_timers_process_events();
 void zjs_timers_init();
+void zjs_timers_free_timers();
```

Comment:
The name is kind of awkward. Maybe zjs_timers_cleanup? Also, add a comment above it here describing what it does - stops and frees all timers. We should move toward @jprestwo's method of describing the APIs here in their header files and using jsdoc format.